api/WebSocket.json

```diff
@@ -0,0 +1,732 @@
+{
+  "api": {
+    "WebSocket": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "11",
+              "notes": "See <a href='https://bugzil.la/695635'>Bug 695635</a>."
+            },
+            {
+              "version_added": "7",
+              "version_removed": "11",
+              "prefix": "Moz",
+              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>Bug 711205</a>)."
+            },
+            {
+              "version_added": "4",
+              "version_removed": "6",
+              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>Bug 711205</a>)."
+            }
+          ],
+          "firefox_android": [
+            {
+              "version_added": "14",
+              "notes": "See <a href='https://bugzil.la/695635'>Bug 695635</a>."
+            },
+            {
+              "version_added": "7",
+              "version_removed": "14",
+              "prefix": "Moz",
+              "notes": "Message size limited to 16 MB (see <a href='https://bugzil.la/711205'>Bug 711205</a>)."
+            }
+          ],
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": null
+          },
+          "safari": {
+            "version_added": true
+          },
+          "safari_ios": {
+            "version_added": true
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "WebSocket": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/WebSocket",
```

Comment:
Incorrect URL

Reply:
Good catch as usual, fixed now in df5f7da.